### PR TITLE
Fix secrets panel load and add run suffix as option

### PR DESCRIPTION
**Pull request by jcenturion**

### CODE REVIEW

File: src/components/editor/secretsEditor.js [hunk 1]
```diff
@@ -8,26 +8,32 @@ export default class A0SecretsEditor extends React.Component {
     constructor(props) {
         super(props);
 
-        const secrets = Object.keys(props.secrets)
+        this._loadSecrets();
+
+        this.keyRefs = [];
```
Comment (ggoodman) [inline src/components/editor/secretsEditor.js]: Can you dump this and the next line as well? They are no longer relevant it seems.

**Comment by ggoodman** [comment]
Why is the secrets fix that you propose needed? Is it because the secrets panel gets a reference to a secrets object that changes later on? If that is the case, a better approach would be to use the `componentWillReceiveProps` lifecycle hook, wouldn't it?

**Comment by jcenturion** [comment]
@ggoodman already modified the code

**Comment by ggoodman** [comment]
LGTM. Please bump version then good to merge.

**Comment by ggoodman** [comment]
It was already bumped. I missed it. Merging.

**Pull request merged**